dev/tests/api-functional/testsuite/Magento/Sales/Service/V1/ShipmentCreateTest.php

```diff
@@ -43,13 +43,13 @@ public function testInvoke()
                 'qty' => $orderItem->getQtyOrdered(),
                 'additional_data' => null,
                 'description' => null,
-                'entity_id' => null,
+                'entity_id' => 1,
                 'name' => null,
                 'parent_id' => null,
                 'price' => null,
                 'product_id' => null,
                 'row_total' => null,
-                'sku' => null,
+                'sku' => 'simple',
```

Comment:
the same PR should be created to https://github.com/magento/magento2
and please cross-link them

Reply:
done https://github.com/magento/magento2/pull/15974